cve_bin_tool/checkers/openldap.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for openldap
+
+https://www.cvedetails.com/product/755/Openldap-Openldap.html?vendor_id=439
+
+"""
+from . import Checker
+
+
+class OpenldapChecker(Checker):
+    CONTAINS_PATTERNS = [r"ldapsearch ([0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
We can take the duplicated pattern out here now.

Reply:
I have made the required changes.